processor/tailsamplingprocessor/config.go

```diff
@@ -220,6 +220,14 @@ type OTTLConditionCfg struct {
 	SpanEventConditions []string       `mapstructure:"spanevent"`
 }
 
+type DecisionCacheConfig struct {
+	// SampledCacheSize specifies the size of the cache that holds the sampled trace IDs
+	// This value will be the maximum amount of trace IDs that the cache can hold before overwriting previous IDs.
+	// For effective use, this value should be at least an order of magnitude higher than Config.NumTraces.
+	// If left as default 0, a no-op DecisionCache will be used.
+	SampledCacheSize int `mapstructure:"sampled_cache_size"`
```

Comment:
thinking about the next steps: we'd likely add a NonSampledCacheSize, and we'll definitely have users who don't care about sampled/non sampled. Would we then have three possible caches internally?

Reply:
Yes, that's definitely the idea!